engine/crates/parser-openapi/src/graph/mod.rs

```diff
@@ -594,7 +594,7 @@ mod tests {
     fn test_graph_size() {
         // Our graph can end up with a ton of nodes & edges so it's important
         // that they don't get too big.
-        assert!(std::mem::size_of::<Node>() <= 48);
+        assert!(std::mem::size_of::<Node>() <= 88);
```

Comment:
This is because of feature unification. The `preserve_order` feature on serde_json is in the workspace, so serde_json::Value can contain an indexmap, and that's larger than a HashMap.

Reply:
The order changes in JSON in tests have the same cause.